vendor/github.com/minio/minio-go/v6/.travis.yml

```
@@ -0,0 +1,37 @@
+- 1.12.x
+- tip
+
+matrix:
+  fast_finish: true
+  allow_failures:
+  - go: tip
+
+addons:
+  apt:
+    packages:
+      - devscripts
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)